javascripts/2048.js

```diff
@@ -63,8 +63,8 @@ Board.prototype.reorient = function(direction) {
 };
 
 Board.prototype.horizontalReorient = function() {
-  return this.board; // or do we want to modify the board in place?
-};
+  return this.board.slice(); // slice() will make a copy for us.
+}; // or do we want to modify the board in place?
```

Comment:
Hmm. This is a good question. Let's do it how you've set it up here and then just make sure we are consistent throughout. If we do it this way, we can reassign board at the end?